README.md

```diff
@@ -275,6 +275,7 @@ More examples of inference can be found in demo/question-answering/
 Feature extraction in NLP is the task to convert text to dense embeddings.  
 It has gained some traction as a robust way to improve search engine relevancy (increase recall).  
 This project supports models from [sentence-transformers](https://github.com/UKPLab/sentence-transformers).
+If you want to use private models from sentence-transformers, you'll need at least the V2.2.0 version.
```

Comment:
Not sure, regarding the code below you are using `use_auth_token` arg, so if there is no kwargs it will crash if it s not the right version. So it's not if you want, but use version XXX. Plus, more important, do the check after the import, and if not Ok raise an exception with a meaningful message